test/props.test.js

```diff
@@ -0,0 +1,78 @@
+class User extends React.Component {
+  render () {
+    return (
+      <span>User {this.props.index}</span>
+    )
+  }
+}
+
+User.propTypes = {
+  index: React.PropTypes.number.isRequired
+}
+
+class Fixture extends React.Component {
+  render () {
+    return (
+      <div>
+        <ul>
+          <li><User index={1} objectProp={{foo: 'bar'}} /></li>
+          <li><User index={2} /></li>
+        </ul>
+      </div>
+    )
```

Comment:
I think we can simplify this example to something like:

```js
class Fixture extends React.Component {
  render() {
    return (
      <a href="http://google.com" style={{display: 'inline'}}>
        google
      </a>
    )
  }
}
```

So we can then remove `User` all together:

```js
describe('#props', () => {
  describe('([ key, key, key... ])', () => {
    it('passes when the actual matches the expected', (wrapper) => {
      expect(wrapper).to.have.props([ 'index' ])
    }, { render: false })
  })

  // ...
})
```